modules/troubleshooting/pages/index.adoc

```diff
@@ -0,0 +1,170 @@
+include::{mod-loc}attributes.adoc[]
+
+//Category: skupper-troubleshooting
+// Type: assembly
+
+[id="troubleshooting"] 
+= Troubleshooting a {service-network}
+
+[role="system:abstract"]
+Typically, you can create a {service-network} without referencing this troubleshooting guide.
+However, this guide provides some tips for situations when the {service-network} does not perform as expected.
+
+See xref:common-problems[] if you have encountered a specific issue using the `skupper` CLI.
+
+A typical troubleshooting workflow is to check all the sites, and create debug tarballs.
+
+// Type: procedure
+[id="checking-sites"] 
+== Checking sites
+
+[role="system:abstract"]
+Using the `skupper` command-line interface (CLI) provides a simple method to get started with troubleshooting {skupper-name}.
+
+.Procedure
+
+. Check the site status:
++
+--
+----
+$ skupper status --namespace west
+
+Skupper is enabled for namespace "west" in interior mode. It is connected to 2 other sites. It has 1 exposed services.
```

Comment:
```suggestion
Skupper is enabled for namespace "west" in interior mode. It is connected to 2 other sites. It has 1 exposed service.
```
Gotta be consistent on on using numerals or words to depict numbers. Numerals are used in this sentence but "two" is used a few sentences later. See https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words

Reply:
changed later instance 'two'->'2'
Cannot change this text as it's cli output